azure-tools-common/src/main/java/com/microsoft/azure/common/logging/Log.java

```diff
@@ -17,7 +17,7 @@ public static void error(String message) {
     }
 
     public static void info(String message) {
-        logger.info(message);
+        System.out.println(message);
     }
```

Comment:
Because in maven, the info level will be print by default, while in gradle , info will not be print by default, here is a walkaround to show the messages, will finally change to notifyUser later